shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/messaging/MessageReaderAdapter.java

```diff
@@ -91,7 +91,7 @@ private MetaStoreEvent map(SerializableListenerEvent listenerEvent) {
     case ON_INSERT: {
       SerializableInsertEvent insert = (SerializableInsertEvent) listenerEvent;
       builder.partitionColumns(new ArrayList<>(insert.getKeyValues().keySet()));
-      builder.partitionColumns(new ArrayList<>(insert.getKeyValues().values()));
+      builder.partitionValues(new ArrayList<>(insert.getKeyValues().values()));
```

Comment:
is there a test for this?

Reply:
Yes, we were aiming to have an integration test for this but since that's proving a bit much work I think it would be good to have a unit test to cover this bug we found.